codes/ruby/selection_sort.rb

```diff
@@ -0,0 +1,28 @@
+=begin
+File: selection_sort.rb
+Created Time: 2024-04-02
+Author: Cy (9738314@gmail.com)
+=end
+
+### 选择排序 ###
+def selection_sort(nums)
+  n = nums.length()
+  # 外排序：未排序区间为[i, n-1]
+  for i in 0...n - 1
```

Comment:
We should write the range in for loop as the following for better readability: 

```suggestion
  for i in 0...(n - 1)
```